config-default.yml

```
@@ -269,7 +269,7 @@ filter:
     notify_user_domains:     false
 
     # Filter configuration
-    ping_everyone:             true  # Ping @everyone when we send a mod-alert?
+    ping_moderators:             true  # Ping @everyone when we send a mod-alert?
```

I guess the comment should be changed as well.
```suggestion
    ping_moderators:             true  # Ping @moderators when we send a mod-alert?
```